src/parser.c

```diff
@@ -268,12 +268,14 @@ parse_tldrlist(char const *path, char const *platform)
     if (sstrncat(fullpath, &len, STRBUFSIZ, platform, strlen(platform)))
         return 1;
 
+    directory = opendir(fullpath);
+    if (directory == NULL)
+        return 1;
```

Comment:
For this return (and the ones above), is the user given an appropriate error message, or does it just silently exit with error code `1`?

Reply:
It actually currently silently exits with error code `0`, which is of course totally wrong.

Reply:
Hm, that's suprising as this is bubbling up to

https://github.com/tldr-pages/tldr-c-client/blob/2adb81b32978dacc2ddd6e0f487d6848a215cb8c/src/tldr.c#L148-L149

which should then returning `EXIT_FAILURE` to the caller?

Reply:
Ah, gotcha. Would you be able to just add a `fprintf(stderr, ...)` output here complaining something about the directory path?